libs/cdk/template/src/lib/template-manager.ts

```diff
@@ -138,79 +142,116 @@ export function createTemplateManager<
   const getContext = notificationKindToViewContext(
     templateSettings.customContext || ((v) => {})
   );
+  let withSuspense = false;
 
   return {
-    addTemplateRef: templates.add,
+    addTemplateRef: (name: N, templateRef: TemplateRef<C>) => {
+      // @ts-ignore
```

Comment:
Do you remember why we need this here?

Reply:
no, I didn't understand and gave up after 5 minutes :D